stdlib/src/test/scala/chisel3/std/BarrelShifter.scala

```diff
@@ -0,0 +1,98 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package chisel3.std
```

Comment:
I think this should belongs to `chisel3.stdTests` package

Reply:
No. Please let's just keep tests in the same package. We want to be able to unittest test package private functions. The whole `chiselTests` package thing was a misguided idea imho.

Reply:
got it!